Changelog.md

```diff
@@ -2,6 +2,8 @@
 
 Language Features:
  * General: Add unit denomination ``gwei``
+ * Yul: Support ``linkersymbol`` builtin in standalone assembly mode.
+ * Yul: Support using string literals exceeding 32 bytes as literal arguments for builtins.
```

Comment:
Not sure if this really needs to be in the changelog, but no harm either I guess and maybe even better to have it.

Reply:
Yeah, it's probably not all that important but still, the limitation is mentioned in the docs so I thought a changelog entry would be warranted.

@chriseth Should I remove it?

Reply:
I think it's good to clarify this here.